frame/staking/src/pallet/mod.rs

```diff
@@ -194,6 +194,7 @@ pub mod pallet {
 
 		/// WIP: This is a noop as of now, the actual business logic that's described below is going
```

Comment:
@kianenigma here the situation is explained, not sure why this did not propagate to the previous diff :/